common/lib/xmodule/xmodule/js/spec/video/video_quality_control_spec.js

```diff
@@ -37,14 +37,6 @@
                 });
             });
 
-            it('bind the quality control', function () {
```

Comment:
I wish it was easier to see the commit messages on changes in a PR.

The actual functionality of the handlers is tested below, and toHandle doesn't work after JQuery update (at least for prototype functions).

There were a host of Jasmine tests (mostly written by the Blades team) where the test both verified that a handler is registered for an event, and then later tested the functionality of the method called when the event fires. All we care about is the functionality when "click" or "play" is done, so this type of test is redundant/not useful.